index.js

```diff
@@ -188,8 +189,17 @@ function initDirectories (done) {
   mkdirp.sync(argv.datadir)
   styles.unpackIfNew(userDataPath, function (err) {
     if (err) logger.error('[ERROR] while unpacking styles:', err)
+    var customLocation = path.join(userDataPath, 'presets', 'default')
+    fs.stat(path.join(customLocation, 'metadata.json'), function (err) {
```

Comment:
This will unpack if there is not a metadata.json because otherwise it might override it.